types/node-fetch/index.d.ts

```diff
@@ -14,6 +14,12 @@
 import { Agent } from "http";
 import { URLSearchParams } from "url";
 
+// `Event` and `AbortSignal` are defined here to prevent a dependency on a particular implementation
+// like the `abort-controller` package, and to avoid requiring the `dom` library
+// in `tsconfig.json`.
+import { AbortSignal } from "./externals";
```

Comment:
We don't need to import the definitions, as we're including them in `tsconfig.ts`.

Reply:
Changed.